plugin/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -30,4 +30,7 @@ grant {
 
     // aws credential file access
     permission java.io.FilePermission "<<ALL FILES>>", "read";
+
+    // AWS credentials needed for clients
+    permission java.io.FilePermission "${user.home}/.aws/-", "read";
```

Comment:
The one you're adding is correct.  The previous one `<<ALL FILES>>` is actually more broad and would match, but should be removed in preference to the one you're adding (Principle of least privilege)